src/components/comment/ActionDropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Media from '../media/Media';
+import { MenuItem } from '../menu';
+import Checkmark16 from '../../icon/fill/Checkmark16';
+import Pencil16 from '../../icon/fill/Pencil16';
+import Trash16 from '../../icon/fill/Trash16';
+import messages from './messages';
+
+import './ActionDropdown.scss';
+
+export interface ActionDropdownProps {
+    commentId: string;
+    onDelete?: (commentId: string) => void;
+    onEdit?: (commentId: string) => void;
+    onResolve?: (commentId: string) => void;
+}
+
+function ActionDropdown({ commentId, onDelete, onEdit, onResolve }: ActionDropdownProps) {
+    if (!onDelete && !onEdit && !onResolve) {
+        return null;
+    }
+
+    return (
+        <Media.Menu data-testid="Comment-actionDropdown" className="bdl-Comment-actionDropdown">
+            {onResolve && (
+                <MenuItem data-testid="ActionDropdownItem-resolve" onClick={() => onResolve(commentId)}>
```

Comment:
Do we need `data-testid` attributes for each of these elements? It seems like we should be able to select them based on their user-facing text.

Reply:
That's right. I've removed `data-testid` attributes and updated tests.